tutorials/jdbcIngest/src/test/resources/example.sql

```diff
@@ -0,0 +1,23 @@
+DROP DATABASE testDev;
```

Comment:
Causes an error if DB is not present, so add if exists clause:
DROP DATABASE IF EXISTS testDev;